lib/task_bunny/publisher.ex

```diff
@@ -21,8 +21,11 @@ defmodule TaskBunny.Publisher do
     e in [ConnectError, PublishError] -> {:error, e}
   end
 
+  @timeout 10_000
```

Comment:
can you rename this to poolboy_timeout and move this under alias definition on top of the file?